ci/requirements.py

```diff
@@ -6,9 +6,12 @@
 from pathlib import Path
 
 try:
-    import tomllib
```

Comment:
For the record, you could also do `import tomli as tomllib` if you preferred to keep the original style.

Reply:
Absolutely, but I saw it somewhere in conda using toml_load and took the opportunity to change it.